ServerSide/yoty/Core/Managers/Suppliers/SuppliersManager.cs

```diff
@@ -26,22 +28,19 @@ public Task<Response<SupplierDTO>> CreateSupplier(NewSupplierRequest newSupplier
 
         public async Task<Response> DeleteSupplier(string supplierId)
         {
-            var supplier = await _context.Suppliers.FindAsync(supplierId).ConfigureAwait(false);
+            var supplier = await _context.Suppliers.Where(s => s.Id == supplierId).Include(b => b.CurrentProposals).FirstOrDefaultAsync().ConfigureAwait(false);
```

Comment:
why we are not deleting the supplier by using is ID? why we should extract it first ? please check if this extraction necessary its not make sense to me that we need to pass the whole entity for deleting its row from the db ... the id should be enough 

here and in all the other places we are deleting entities from the tables